inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongStreamController.java

```diff
@@ -169,4 +176,10 @@ public Response<Boolean> delete(@RequestParam String groupId, @RequestParam Stri
         return Response.success(streamService.delete(groupId, streamId, username));
     }
 
+    @RequestMapping(value = "/stream/parseFields", method = RequestMethod.POST)
+    @ApiOperation(value = "parse inlong stream fields")
```

Comment:
1. "parse inlong stream fields" -> "Parse inlong stream fields from JSON string"
2. Add `@ApiImplicitParam` for `fieldsJson`.